visitor/models.py

```diff
@@ -1,26 +1,60 @@
 from django.db import models
 from django.utils.timezone import datetime
+from syllabus import models as syllabus_models
 
+gender_choice = (
+    (True, 'Male'),
+    (False, 'Female'),
+)
 
-class Enroll(models.Model):
-    first_name = models.CharField(max_length = 300, null=False, blank=False, default="John")
-    middle_name = models.CharField(max_length = 300, null=True, blank=True)
-    last_name = models.CharField(max_length = 300, null=False, blank=False, default="Doe")
-    email = models.EmailField(max_length=70,blank=False, null= False, unique= True)
-    gender = models.BooleanField(default=True) #True=Male
-    education = models.CharField(max_length=800, null=False, blank=False, default="Incwell Bootcamp")
-    phone = models.CharField(max_length=500, null=False, blank=False, default="9800000")
-    git_link = models.CharField(max_length=800, null=True, blank=True, default="No Git link")
+class Student(models.Model):
+    firstName = models.CharField(max_length = 300, null=False, blank=False)
+    middleName = models.CharField(max_length = 300, null=True, blank=True)
+    lastName = models.CharField(max_length = 300, null=False, blank=False)
+    email = models.EmailField(max_length=70,blank=False, null= False, unique= True) 
+    gender = models.BooleanField(max_length=1, choices=gender_choice, null=False, blank=False)
+    education = models.CharField(max_length=800, null=False, blank=False)
+    phone = models.CharField(max_length=500, null=False, blank=False)
+    gitLink = models.CharField(max_length=800, null=True, blank=True)
+    course = models.ManyToManyField(syllabus_models.Course, related_name="student_course")
+
+    def __str__(self):
+        return f'{self.firstName} {self.middleName} {self.lastName}'
 
+    def save(self, *args, **kwargs):
+        student = super(Student, self).save(*args, **kwargs)   
+        
 
+class Enroll(models.Model):
+    student = models.ManyToManyField(Student, related_name="enroll_student")
+    course = models.ManyToManyField(syllabus_models.Course, related_name="enroll_course")
+    
     def __str__(self):
-        return f'{self.first_name} {self.middle_name} {self.last_name}'
+        return f'{self.student}'
+        
+# def save_post(sender, instance, **kwargs):
```

Comment:
if these commented codes are important to save?

if yes write a comment above it, why is so?

Reply:
git can track your history. there is no need to keep commented code we can get it from history.